### PR TITLE
Don't display text questions that haven't been answered

### PR DESCRIPTION
**Pull request by hampelm**

### CODE REVIEW

**Comment by hampelm** [comment]
Checking one thing before this is ready.

File: src/js/views/responses/item.js [hunk 1]
```diff
@@ -126,6 +126,11 @@ function($, _, Backbone, events, settings, util, api, Responses, template) {
         } else if (question.type === 'text') {
```
Comment (hampelm) [inline src/js/views/responses/item.js]: Do we even need this block? Answered text questions are picked up by the block above. Is it used for legacy response formats?

Comment (prashtx) [inline src/js/views/responses/item.js]: This came in with the change for editing text fields (2ccf87a8c0a47fa8cf8d994dca8f14952490d1b6). Was the intention to allow adding notes if the original response had no notes? If so, we may need to allow empty text fields here and prevent display in the template (which has different modes for read-only display mode and edit-mode display, as i recall).

**Comment by hampelm** [comment]
That's it! I'll redo this change to hide text entries with no data in the template, but still make them editable.

**Pull request closed without merging**